lib/system.rb

```diff
@@ -84,6 +84,12 @@ def delete_file!(path_from_root)
     logger.success("Deleted `#{path_from_root}`")
   end
 
+  def delete_files!(directory_path_from_root)
+    logger.debug("→ Deleting files under directory `#{directory_path_from_root}`")
+    FileUtils.rm_rf(Dir.glob("#{directory_path_from_root}/*"))
```

Comment:
Please use `path_for` for everything in this class (ensures we're scoped properly).

Reply:
Done